src/docs/product/releases/index.mdx

```diff
@@ -201,182 +130,44 @@ When Sentry sees this commit, we’ll reference the commit in the issue, and whe
 
 <Note>
 
-If you’re using GitHub, you may have a privacy setting enabled which prevents Sentry from identifying the user’s real email address. If you wish to use the suggested owners feature, you’ll need to ensure “Keep my email address private” is unchecked in GitHub’s [account settings](https://github.com/settings/emails).
+If you’re using GitHub, you may have a privacy setting enabled that prevents Sentry from identifying the user’s actual email address. To use the suggested owners feature, uncheck “Keep my email address private” in GitHub’s [account settings](https://github.com/settings/emails).
 
 </Note>
 
-#### Alternatively: Without a Repository Integration
-
-If you don’t want Sentry to connect to your repository, or you’re using an unsupported repository provider or VCS (e.g. Perforce), you can tell Sentry about your raw commit metadata via the CLI or the API.
-
-##### Using the CLI
-
-```bash
-# Assumes you're in a git repository
-export SENTRY_AUTH_TOKEN=...
-export SENTRY_ORG=my-org
-VERSION=$(sentry-cli releases propose-version)
-
-# Create a release
-sentry-cli releases new -p project1 -p project2 $VERSION
+#### Finalize the Release
 
-# Associate commits with the release
-sentry-cli releases set-commits --auto $VERSION
-```
+By default, a release is created as “unreleased”. Finalizing a release means that we populate a second timestamp on the release record, which is prioritized over `date_created` when sorting releases in sentry.io. Release finalization (and the timestamp) also affects:
+- What counts as "the next release" for resolving issues
+- Which release is used as the base for associating commits if you use `--auto`
```

Comment:
This is not entirely clear. If I finalize my release now, what does that do to associating commits? Also note that `--auto` refers to a CLI option on a specific command.

Reply:
Thanks, Jan. This was from the original draft. @jonesphillip - can you please clarify.

Reply:
@jonesphillip - can you clarify?